commands/show_cluster.go

```diff
@@ -250,6 +250,10 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 
 	output = append(output, color.YellowString("Kubernetes API endpoint:")+"|"+clusterDetails.APIEndpoint)
 
+	if len(clusterDetails.AvailabilityZones) > 0 {
+		output = append(output, color.YellowString("Availability Zones:")+"|"+strings.Join(clusterDetails.AvailabilityZones, ","))
```

Comment:
Pretty straight forward, but no in which order we want this magical output to be.